### DISCUSSION TITLE
Is there a way to convert "SingleFileZ" file to "SingleFile" file?

### DISCUSSION THREAD

**Discussion by igorlima1701** [gildas-lormeau/SingleFileZ#140] [Q&A] [answered]
I prefer archiving a web page with SingleFileZ in my HD. 

But when I want to share a saved web page with someone I think that its best to share a page that was created with SingleFile since it can be opened instantly in any device.

So, without having to go to the original online web page since it can no longer exist or could be changed, Is there a way to convert "SingleFileZ" file to "SingleFile" file for those times that I want to share some web page that is in my archives?

Is it ok to use the "SingleFile" extension on a "SingleFileZ" file opened in browser? 

Thank you for those extensions!

**Comment by gildas-lormeau** [comment, marked as answer]
You should be able to save in the browser with SingleFile a page saved SingleFileZ. If that does not work or if you prefer to do this from the command line (Linux, WSL, macOS, BSD etc.), you can follow this procedure: https://gist.github.com/gildas-lormeau/f05b65a8654587ea89857c0213d809a9.

**Reply by igorlima1701** [reply]
When I try to save in the browser with SingleFile a page saved SingleFileZ I get some erros. And unfortunately I don't know how to use command line. But this is no big deal. 
Thank you for the response and thank you again for those extensions!

**Reply by gildas-lormeau** [reply]
If you're using Chrome or a Chromium-based browser, make sure that "Allow access to file URLs" is enabled in the extension page of SingleFile, e.g. chrome://extensions/?id=mpiodijhokgodhhofbcjdecpffjipkle in Chrome.

**Reply by igorlima1701** [reply]
Thank you for fast the reply!

Indeed I'm using Chrome and that option is enabled it. 

But for exemple: This [page  ](https://letterboxd.com/journal/swell-superbia-the-supporting-men-of-tick-tick-boom/) after I save it with SingleFileZ  and then save it with SingleFile and open it, after some seconds the page becomes like this: 


![2022-10-22 22_26_12-](https://user-images.githubusercontent.com/69910018/197368774-d1282390-26dc-4766-afa3-afa4f60e0fc8.png)

**Reply by gildas-lormeau** [reply]
Indeed, that's an unfortunate conflict with SingleFileZ and can be considered as a bug. If you disable SingleFileZ, you will be able to view the page saved with SingleFile. I'll try to fix this bug in the next version. Otherwise, it works well in Firefox with both extensions enabled.

Edit: I confirm the fix will be available in the next version of SingleFileZ. The pages you already "re-saved" with SingleFile will also work without needing to disable SingleFileZ once the new version published, i.e. in a couple of days for Chrome.

**Reply by igorlima1701** [reply]
Yes! when SingleFileZ is disabled, the SingleFile file (produced from the original SingleFileZ file) doesn't show the "please wait" page anymore in chrome.

It's really good to know that it's possible to make this conversion! 

And excellent to know about the fix for chrome.

Thank you!

**Reply by gildas-lormeau** [reply]
You're welcome! FYI, the version 1.3.33 which fixes this issue is available for download.
